commands.json

```
@@ -529,6 +529,42 @@
     "since": "5.0.0",
     "group": "sorted_set"
   },
+  "BZMPOP": {
+    "summary": "Remove and return members with scores in a sorted set or block until one is available",
+    "complexity": "O(K) + O(N*log(M)) where K is the number of provided keys, N being the number of elements in the sorted set, and M being the number of elements popped.",
+    "arguments": [
+      {
+        "name": "timeout",
+        "type": "double"
+      },
+      {
+        "name": "numkeys",
+        "type": "integer"
+      },
+      {
+        "name": "key",
+        "type": "key",
+        "optional": true,
+        "multiple": true
+      },
```

Since the number of keys must be gt 0, the first key is actually mandatory.

```suggestion
      {
        "name": "key",
        "type": "key",
      },
      {
        "name": "key",
        "type": "key",
        "optional": true,
        "multiple": true
      },
```